applications/RomApplication/custom_strategies/rom_builder_and_solver.h

```diff
@@ -229,7 +229,12 @@ class ROMBuilderAndSolver : public BuilderAndSolver<TSparseSpace, TDenseSpace, T
     SizeType GetNumberOfROMModes() const noexcept
     {
         return mNumberOfRomModes;
-    } 
+    }

```

Comment:
By only exporting this to python, we are less intrusive in the RomB&S and no new variable should be created